Makefile

```diff
@@ -15,10 +15,14 @@ OSD_E2E_TEST_HARNESS_IMG=$(IMAGE_BASE)-test-harness:$(VERSION)
 OSD_E2E_TEST_HARNESS_IMG_LATEST=$(IMAGE_BASE)-test-harness:latest
 
 # running `make` builds the operator (default target)
-all: operator
+.DEFAULT_GOAL := operator
```

Comment:
Afaiu the `.DEFaULT_GOAL` variable is intended for more complex use cases. The make docs say

> The .DEFAULT_GOAL variable allows you to discover the current default goal, restart the default goal selection algorithm by clearing its value, or to explicitly set the default goal.

I don't a benefit over just letting make pick the first target as the default.

Reply:
The advantage of `.DEFAULT_GOAL` is that you explicitly set the default goal instead of relying on the structure of the Makefile to define the default goal. For example, in this Makefile we `include` another file above this declaration, which already defines some goals of which one becomes the default goal.

We can either move the goal we want as default further up in the file (above the `include`, in this case) or define what we want explicitly (which is what I opted for).

Reply:
Fair point, I didn't think of the import.